packages/blockchain-wallet-v4/src/redux/payment/btc/sagas.js

```diff
@@ -368,6 +368,17 @@ export default ({ api }) => {
   }
 
   return {
-    create: create
+    create,
+    calculateAmount,
+    calculateEffectiveBalance,
+    calculateFee,
+    calculateFrom,
+    calculatePublish,
+    calculateTo,
+    calculateSelection,
+    calculateSignature,
+    calculateSweepSelection,
+    getWalletUnspent,
+    pushBitcoinTx
```

Comment:
are these supposed to exposed and used publicly?  if not, maybe prefix the exports with `__[methodName]` to indicate they should not be called directly

Reply:
Good call, prefix added